app/_locales/en/messages.json

```
@@ -706,7 +706,7 @@
     "message": "Gas limit is the maximum amount of units of gas you are willing to spend."
   },
   "gasLimitTooLow": {
-    "message": "Gas limit must be at least 21000"
+    "message": "Gas limit must be at least $1"
```

Nit: It would be nice to add a description here
```suggestion
    "message": "Gas limit must be at least $1",
    "description": "$1 is the custom gas limit, in decimal."
```